.cqfd/docker/Dockerfile

```diff
@@ -36,7 +37,6 @@ RUN set -x \
     && sed -i "s/# en_US\.UTF-8 UTF-8/en_US\.UTF-8 UTF-8/" /etc/locale.gen \
     && locale-gen \
     && dpkg-reconfigure locales \
-    && pip install "ansible>=2.9,<2.10,!=2.9.10" \
```

Comment:
Ceph-ansible 6.0 only supports Ansible 2.10: https://docs.ceph.com/projects/ceph-ansible/en/latest/#releases
We need to keep the Ansible version check.

Reply:
The apt version is Ansible 2.10.
I have removed the pip install and replaced it by apt version.

Reply:
why not use directly the Dockerfile we use in debiancentos? if the idea is to merge in the end...
https://github.com/seapath/ansible/blob/debiancentos/.cqfd/docker/Dockerfile

Reply:
Good point, I have made the change.